### ISSUE TITLE
Cannot login into App store on Catalina

### ISSUE THREAD

**Issue by mogupta** [RehabMan/OS-X-Null-Ethernet#10]
Hi 

I tried the DSDT method on Catalina. An ethernet interface is available in the network settings but when I sign into App store, it doesn't sign it. Tried multiple times. 

Please help 
Mohit

**Comment by mikemike396** [comment]
I got it working using the SSDT method (Specifically with Clover).

Couple things to note.

1. After installing in "Place in /EFI/CLOVER/ACPI/patched/ssdt-X.aml where 'X' is some number that you're not already using for SSDTs.", be sure to reboot.

2. After reboot, go to System Preferences -> Network. Change location to "Automatic" (just in case you have different locations stored) and remove all of your network adapters using the "-" button located below the list of adapters.

3. Go to /Library/Preferences/System Configuration and remove NetworkInterfaces.plist.

4. Reboot

5. Go to System Preferences -> Network and manually add the adapter to the Ethernet Adapter FIRST, then add your WIFI adaptor.

I was able to log into the App Store after doing this. Prior to this it would just dismiss the login window but never log in. Hope this helps.
